src/Entity/OgMembership.php

```diff
@@ -93,21 +94,50 @@ public function setCreatedTime($timestamp) {
   /**
    * {@inheritdoc}
    */
-  public function getUser() {
+  public function getOwner() {
     return $this->get('uid')->entity;
   }
 
   /**
    * {@inheritdoc}
    */
-  public function setUser(AccountInterface $user) {
-    $this->set('uid', $user->id());
+  public function getOwnerId() {
+    return $this->get('uid')->target_id;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setOwner(UserInterface $account) {
+    $this->set('uid', $account->id());
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setOwnerId($uid) {
+    $this->set('uid', $uid);
     return $this;
   }
 
   /**
    * {@inheritdoc}
    */
+  public function getUser() {
+    return $this->getOwner();
+  }
```

Comment:
It's a bit pointless to have two methods with the same functionality. Shall we mark these as deprecated, or (since we are not even in alpha yet) remove them altogether?